ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -85,7 +85,12 @@ final class Runner[T <: ReadWriteService, Extra](
     )
     implicit val materializer: Materializer = Materializer(actorSystem)
 
-    val sharedEngine = new Engine(EngineConfig(config.allowedLanguageVersions))
+    val sharedEngine = new Engine(
+      EngineConfig(
+        config.allowedLanguageVersions,
+        forbidV0ContractId = true,
```

Comment:
we forbid V0 Contract ID in KV.

Reply:
I think we might need to set it somewhere for pkvutils separately. I’m not sure it goes via this runner. Maybe we should turn on the rejectv0 tests by default to makes sure we notice if we forget to do this in pkvutils.

Reply:
@miklos-da, FYI.